src/compute/src/sink/persist_sink.rs

```diff
@@ -58,7 +62,13 @@ where
     where
         G: Scope<Timestamp = Timestamp>,
     {
-        let desired_collection = sinked_collection.map(Ok).concat(&err_collection.map(Err));
+        let mut desired_collection = sinked_collection.map(Ok).concat(&err_collection.map(Err));
+
+        // If `REFRESH EVERY` was specified, round up timestamps.
```

Comment:
Some of the comments in this commit explicitly mention `REFRESH EVERY`, but there are other `REFRESH` options to which they apply as well, right?

Reply:
Right!